CHANGELOG.md

```diff
@@ -4,6 +4,8 @@ Changelog
 **Unreleased**
 --------------
 
+- Add blank behavior parameters to `PropertyResolver.optionalStringProvider` overloads to specify behavior for blank property values. Default behavior now is to error. 
```

Comment:
I think CI check is failing because there needs to be `spotlessApply` on here?

Reply:
yup fixing